src/sidebar/helpers/query-parser.ts

```diff
@@ -18,20 +23,41 @@ const filterFields = [
   'user',
 ];
 
+/**
+ * Names of fields that can be used in `{field}:{term}` queries with
+ * {@link parseHypothesisSearchQuery}.
+ */
+export type SearchField = 'group' | 'quote' | 'tag' | 'text' | 'uri' | 'user';
+
+const searchFields: SearchField[] = [
+  'group',
+  'quote',
+  'tag',
+  'text',
+  'uri',
+  'user',
+];
```

Comment:
Perhaps it would be useful to infer `SearchField` type from `searchFields` array and `SearchField` from `filterFields` array. That way we don't have to remember adding the values in two places.

```ts
const filterFields = [
  'cfi',
  'quote',
  'page',
  'since',
  'tag',
  'text',
  'uri',
  'user',
] as const;

export type FilterField = typeof filterFields[number];
```

(I'm assuming `filterFields` and `searchFields` does not need to change at runtime and can be defined `as const`)

Reply:
> I'm assuming filterFields and searchFields does not need to change at runtime and can be defined as const

That's correct. I made this change. Thanks!